app/javascript/components/common/MarkdownEditor.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import SimpleMDE from 'react-simplemde-editor'
+import 'easymde/dist/easymde.min.css'
+import { sendRequest } from '../../utils/send-request'
+import { useIsMounted } from 'use-is-mounted'
+
+export const MarkdownEditor = ({ uuid }: { uuid: string }): JSX.Element => {
```

Comment:
I was initially wondering why the markdown editor needs a uuid :D

Reply:
Why does it? :)